examples/java/src/main/java/org/apache/beam/examples/WindowedWordCount.java

```diff
@@ -211,19 +178,61 @@ public static void main(String[] args) throws IOException {
     PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.CountWords());
 
     /**
-     * Concept #5: Format the results for a BigQuery table, then write to BigQuery.
-     * The BigQuery output source supports both bounded and unbounded data.
+     * Concept #5: Key by the window, so we can write one file per window. To access the window in a
```

Comment:
How about something that generalizes and then gets specific?

At this point, the data is organized by window and can be adapted for the specific output format. We're using text files here for simplicity so we rewindow and...